app/assets/javascripts/license_form.js

```diff
@@ -15,4 +15,32 @@ $(document).ready(function() {
       $('#media_object_terms_of_use_custom').addClass('hidden');
     }
   });
+
```

Comment:
Below the indentation looks inconsistent -- are tabs getting mixed with spaces?
Seems the file was using two spaces for indentation.

Reply:
Changed.